src/packages/xroad/debian/changelog

```diff
@@ -1,3 +1,16 @@
+xroad (6.14.0) trusty; urgency=low
+
```

Comment:
The software version is still 6.14.0. The "1" part is the packaging version, which is a .deb (and rpm) convention we have been following. A .deb package version number consist of the upstream software version (here 6.14.0) and the packaging version (here 1), separated by a hyphen.